webhook/configmaps/configmaps.go

```diff
@@ -58,13 +61,19 @@ type reconciler struct {
 }
 
 var _ controller.Reconciler = (*reconciler)(nil)
+var _ pkgreconciler.LeaderAware = (*reconciler)(nil)
 var _ webhook.AdmissionController = (*reconciler)(nil)
 var _ webhook.StatelessAdmissionController = (*reconciler)(nil)
 
 // Reconcile implements controller.Reconciler
 func (ac *reconciler) Reconcile(ctx context.Context, key string) error {
 	logger := logging.FromContext(ctx)
 
+	if !ac.IsLeaderFor(types.NamespacedName{Name: ac.name}) {
```

Comment:
Namespace?

Reply:
This is reconciling the validating webhook, so no namespace.

Reply:
we're really abusing the type, whose comment says `mandatory namespace` 😁